config/cluster-install/rbac/numaflow-cluster-role.yaml

```diff
@@ -42,6 +43,14 @@ rules:
       - update
       - patch
       - delete
+  - apiGroups:
+    - metrics.k8s.io
```

Comment:
Is it ok to re-use the existing cluster role?

Reply:
No.

Reply:
Done